modelator_py/apalache/cli.py

```diff
@@ -24,7 +24,7 @@ def pure(self):
             self._stdin is not None
         ), "The pure interface requires json input in stdin"
         json_dict = stdjson.loads(self._stdin.read())
-
+        
```

Comment:
Did you run the hooks? I think the formatter would not have allowed this whitespace.